bg-bg/logtalk-bg.html.markdown

```diff
@@ -0,0 +1,592 @@
+---
+language: Logtalk
+contributors:
+    - ["Paulo Moura", "http://github.com/pmoura"]
+translators:
+    - ["vsraptor", "https://github.com/vsraptor"]
+lang: bg-bg
+
```

Comment:
Could you follow the standard here? For reference [https://github.com/adambard/learnxinyminutes-docs/blob/master/bg-bg/perl-bg.html.markdown](https://github.com/adambard/learnxinyminutes-docs/blob/master/bg-bg/perl-bg.html.markdown).

Reply:
@prertik What seems to be the issue here?

Reply:
Well, it's missing the format. Following should be the format, taken from the reference:
```md
name: perl
category: language
language: perl
filename: learnperl-bg.pl
contributors:
    - ["Korjavin Ivan", "http://github.com/korjavin"]
    - ["Dan Book", "http://github.com/Grinnz"]
translators:
    - ["Красимир Беров", "https://github.com/kberov"]
lang: bg-bg
```
Additionally, there's no downloadable file. Why?